src/components/Compare/CompareSelector.tsx

```diff
@@ -0,0 +1,46 @@
+import { Button, Grid } from "@trussworks/react-uswds";
+import { Dispatch, SetStateAction } from "react";
+import { CompareProviders } from "../../pages/Compare";
+import Selections from "./Selections";
+
+type CompareSelectorProps = {
+  providers: CompareProviders;
+  setProviders: Dispatch<SetStateAction<CompareProviders>>;
+};
+
+function CompareSelector({ providers, setProviders }: CompareSelectorProps) {
+  if (!providers.providerA && !providers.providerB) return <></>;
+  return (
+    <div className="bg-primary-darker width-full padding-y-4 padding-x-2">
+      <Grid row>
+        <Grid col={12} tablet={{ col: 6 }}>
```

Comment:
would the spacing look ok if tablet had more than 6 and desktop was 6? it looks great on desktop, but at sizes slightly larger than mobile, the compare pane gets cramped and takes up a lot of vertical space b/c the names are so long

Reply:
Makes sense, will do!